components/NavigationDrawer/src/MDCBottomDrawerViewController.h

```diff
@@ -106,6 +109,17 @@
  */
 @property(nonatomic, weak, nullable) id<MDCBottomDrawerViewControllerDelegate> delegate;
 
+/**
+ This block is called after a change of the bottom drawer view controller's elevation or one of its
+ view hierarchy ancestors. Use this block to respond to elevation changes in the view or its
+ ancestor views.
+ @param elevation The @c mdc_currentElevation plus the @c mdc_currentElevation of all ancestor
+ views.
+ @param object This bottom drawer view controller.
+ */
+@property(nonatomic, copy, nullable) void (^mdc_elevationDidChangeBlock)
+    (MDCBottomDrawerViewController *_Nonnull object, CGFloat elevation);
```

Comment:
`object` should be changed to `bottomDrawerViewController`

Reply:
done.